### PR TITLE
Creating map

### PR DESCRIPTION
**Pull request by ghost**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## What does this PR do?
This PR adds in the map functionality that will be visible from the User's dashboard once they login/start the application. This PR also adds in hard-coded data (which will change to a fetch call from the BE once their endpoints are set up) and then maps over the data to place markers/pins on the map in the appropriate location. When a pin is selected, it will also display the type of activity, as well as any notes (this part can be changed to display different information if needed). This PR also adds functionality that will ask for a User's location upon start of the app to accurately identify where the map needs to be centralized in relation to the users position.
## Related Tickets & Documents

This PR will close issue #3 and also issue #4 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

Screen Shot 2020-04-07 at 2 32 49 PM

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: components/Map/Map.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import { StyleSheet, Text, View, Button, TextInput } from 'react-native';
+
+export default function Map() {
+
+let events = [
+    {
+      id: 1,
+      date: 20200407,
+      time: 1700,
+      unix_time: 1586278800,
```
Comment (ZaneMeroff) [inline components/Map/Map.js:13]: Not sure how the unix time will tie into the BE of our project. I want to discuss this further as a team.

File: components/Map/Map.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    },
+    {
+      id: 2,
+      date: 20200409,
+      time: 1700,
+      unix_time: 1586451600,
+      activity: 'Spikeball',
+      location: 'Cheeseman Park',
+      lat: 39.731,
+      long: -104.973,
```
Comment (ZaneMeroff) [inline components/Map/Map.js:42]: Are we going to be using the longitude and latitude for pin placement on the map? I would like to discuss how this works with you and how we can best implement it for the BE.

File: components/Map/Map.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      unix_time: 1586635200,
+      activity: 'Kick Ball',
+      location: 'Pepsi Center',
+      lat: 39.748,
+      long: -105.009,
+      duration: 'Could be a min and max input from the user -- ex. 15 min 120 max',
+      notes: 'If the weather is poor, meet here instead',
+      max_players: 20,
+      players_going: 10,
+      skill_level: 'Amateur',
```
Comment (ZaneMeroff) [inline components/Map/Map.js:72]: We should also include a "just for fun" type in the skill level selection.

File: package.json [hunk 4]
```diff
@@ -12,6 +12,7 @@
     "react": "~16.9.0",
     "react-dom": "~16.9.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.0.tar.gz",
+    "react-native-maps": "^0.27.1",
```
Comment (ZaneMeroff) [inline package.json:18]: Didn't realize there was a specific maps component for react native, this is perfect!

File: components/Map/Map.js [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+export default function Map() {
+
+let events = [
+    {
+      id: 1,
+      date: 20200407,
+      time: 1700,
+      unix_time: 1586278800,
+      activity: 'Volleyball',
+      location: 'Wash Park',
```
Comment (ZaneMeroff) [inline components/Map/Map.js:15]: We're going to need to find a way to tie names of parks with their specific locations for the map to recognize. Not sure if this happens automatically or if it's something we need to add code to do.

Comment (ghost) [reply components/Map/Map.js:15]: Yeah we will have to discuss that with BE. I believe there's functionality they can add to find the coordinates for places, but i'm not 100% certain, so we can reach out to them about that

File: components/Map/Map.js (hunk 2 above)
Comment (ghost) [reply components/Map/Map.js:42]: Yes that's how we will be placing pins on the map. Hopefully our BE can get us the coordinates when a user selects a location

File: components/Map/Map.js (hunk 3 above)
Comment (ghost) [reply components/Map/Map.js:72]: Yeah for sure! I think limiting these inputs to a drop-down when they create an event will definitely limit how wide of a range they can choose when making events

File: package.json (hunk 4 above)
Comment (ghost) [reply package.json:18]: Yeah super dope 😎

File: components/Map/Map.js (hunk 1 above)
Comment (ghost) [reply components/Map/Map.js:13]: Yeah I think they said for sorting future events and placing those in order, but we can double check with them

**Pull request merged**